include/eve/traits/common_value.hpp

```diff
@@ -0,0 +1,70 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <type_traits>
+
+  //================================================================================================
+  //! @addtogroup traits
+  //! @{
+  //!
+  //! @struct common_value_t
+  //!
+  //! @brief Computes a type that can represent all values in a list of types.
+  //!
+  //! **Required header:** `#include <eve/traits/common_value.hpp>`
+  //!
+  //! Similar to `std::common_type`.
```

Comment:
Don't copy paste documentation, this is not what `common_value` does.

Reply:
ok, I will remove it at the next push